scripts/eval_calc_errors.py

```diff
@@ -15,14 +15,23 @@
 from bop_toolkit_lib import inout
 from bop_toolkit_lib import misc
 from bop_toolkit_lib import pose_error
-from bop_toolkit_lib import pose_error_htt
+
 from bop_toolkit_lib import renderer
 from bop_toolkit_lib import renderer_batch
 
 # Get the base name of the file without the .py extension
 file_name = os.path.splitext(os.path.basename(__file__))[0]
 logger = misc.get_logger(file_name)
 
+htt_available = False
+try:
+    from bop_toolkit_lib import pose_error_htt
+    htt_available = True
+except ImportError as e:
+    logger.warn("""Missing hand_tracking_toolkit dependency, 
```

Comment:
please add instructions here

Reply:
"""Missing hand_tracking_toolkit dependency, mandatory if you are running evaluation on HOT3d. Refer to the README.md for installation instructions." ?

Reply:
Modified message in new commit